test/lib/mayaUsd/nodes/testProxyShapeBase.py

```diff
@@ -396,6 +396,46 @@ def testShareStagePreserveTCPS(self):
         self.assertEqual(stage.GetTimeCodesPerSecond(), tcps)
         self.assertEqual(stage.GetRootLayer().timeCodesPerSecond, tcps)
 
```

Comment:
The functions below were declared repeatedly, inline in multiple tests. Declare them here and document them instead.